replication.proto

```diff
@@ -0,0 +1,131 @@
+
+syntax = "proto3";
+package replication;
+
+import "google/protobuf/descriptor.proto";
+
+option go_package = ".;replication";
+
+extend google.protobuf.FieldOptions {
+  // Indicates that a field MAY contain information that is sensitive
+  // and MUST be treated as such (e.g. not logged).
+  bool replication_secret = 1059;
+}
+
+service Controller {
+  rpc EnableVolumeReplication (EnableVolumeReplicationRequest)
+    returns (EnableVolumeReplicationResponse) {}
+
+  rpc DisableVolumeReplication (DisableVolumeReplicationRequest)
+    returns (DisableVolumeReplicationResponse) {}
+
+  rpc PromoteVolume (PromoteVolumeRequest)
+  returns (PromoteVolumeResponse) {}
+
+  rpc DemoteVolume (DemoteVolumeRequest)
+  returns (DemoteVolumeResponse) {}
+
+  rpc ResyncVolume (ResyncVolumeRequest)
+  returns (ResyncVolumeResponse) {}
```

Comment:
We should keep the same indent in all return statements

Reply:
waiting to see if the CI catches it one :P

Reply:
We need to merge CI for that or the other way around is we can rebase my changes in your PR to have them. This way we can run CI.